composables/masto/account.ts

```diff
@@ -7,10 +7,14 @@ export function getDisplayName(account: mastodon.v1.Account, options?: { rich?:
   return displayName.replace(/:([\w-]+?):/g, '')
 }
 
+export function acctToShortHandle(acct: string) {
+  return `@${acct.includes('@') ? acct.split('@')[0] : acct}`
+}
+
 export function getShortHandle({ acct }: mastodon.v1.Account) {
   if (!acct)
     return ''
-  return `@${acct.includes('@') ? acct.split('@')[0] : acct}`
+  return acctToShortHandle(acct)
 }
```

Comment:
I'm not quite sure why these two were split out - aren't they now doing the same thing?

Reply:
Mostly, but one receieves an account object, and the other one a well defined acct string. In the mentions where I used this, there was no account object